common/app/views/fragments/atoms/mediaEmbed.scala.html

```diff
@@ -16,6 +18,52 @@
     </head>
     <body>
         @views.html.fragments.atoms.media(page.atom, displayCaption = displayCaption)
+ <script>
+        var guardian = {
+            isEmbed: true,
+            isModernBrowser: (
+                'querySelector' in document
+                && 'addEventListener' in window
+                && 'localStorage' in window
+                && 'sessionStorage' in window
+                && 'bind' in Function
+                && (
+                    ('XMLHttpRequest' in window && 'withCredentials' in new XMLHttpRequest())
+                    || 'XDomainRequest' in window
+                )
+            ),
+            config: @Html(templates.js.javaScriptConfig(model.MediaAtomEmbedPage(page.atom)).body),
+            adBlockers: { onDetect: [] }
+
+        };
+        @* Decide the Ophan PV ID here so we can share it with Google Analytics *@
+        guardian.config.ophan = {
+            // This is duplicated from
+            // https://github.com/guardian/ophan/blob/master/tracker-js/assets/coffee/ophan/transmit.coffee
+            // Please do not change this without talking to the Ophan project first.
+            pageViewId: new Date().getTime().toString(36) + 'xxxxxxxxxxxx'.replace(/x/g, function () {
+                return Math.floor(Math.random() * 36).toString(36);
+            })
+        };
+        @* Find the Ophan browser ID as well, for sharing with GA *@
+        @Html(templates.inlineJS.nonBlocking.js.ophanConfig().body)
+        </script>
+
+        <script>
+            @Html(curlConfig().body)
+
+            @if(Configuration.assets.useHashedBundles) {
+                window.curl.paths['bootstraps/enhanced/youtube'] = '@Static("javascripts/bootstraps/enhanced/youtube.js")';
+                window.curl.paths['ophan/embed'] = '@{Configuration.javascript.config("ophanEmbedJsUrl")}';
```

Comment:
don't think version of ophan will get used as youtube-tracking pulls in the full ophan https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/video/youtube-tracking.js#L57